egs/wsj/s5/steps/make_mfcc.sh

```diff
@@ -11,6 +11,7 @@ cmd=run.pl
 mfcc_config=conf/mfcc.conf
 compress=true
 write_utt2num_frames=false  # if true writes utt2num_frames
+out_suffix=
```

Comment:
Please revert this change to make_mfcc.sh and instead of storing the MFCC features in 'mfcc/', store them in subdirectories of the data-dirs.  This can be done by removing the last two arguments from the make_mfcc.sh command line.   This will simplify your life.  Just make sure you don't ever 'mv' data-dirs.